routes/apiHandlers.js

```diff
@@ -1716,10 +1715,9 @@ async function handleCreateAccountNew(req) {
     });
 
     // TOS
-    const tosDate = moment().format('YYYYMMDD');
     const tosParams = [
         decoded.username,
-        `accepted_tos_${tosDate}`,
+        'accepted_tos_20180614',
```

Comment:
what if we change policy, the tosDate will change too, so it is better to keep a variable like tosDate??

Reply:
This name is a fixed name.  https://github.com/steemit/condenser/blob/master/src/server/api/general.js#L12